python/lsst/daf/persistence/repository.py

```diff
@@ -120,7 +120,7 @@ def _initMapper(self, repositoryCfg):
                     if mro[-1] is object:
                         mro = mro[:-1]
                     for c in mro:
-                        if arg in inspect.getfullargspec(c.__init__ ).args:
+                        if arg in inspect.getargspec(c.__init__ ).args:
```

Comment:
I guess this could get squashed out? (it doesn't matter much to me personally. I just noticed it)